### DISCUSSION TITLE
Wazuh Automated Deployment via forward node

### DISCUSSION THREAD

**Discussion by wildlyunder** [Security-Onion-Solutions/securityonion#6132] [Unsupported Versions] [answered]
Hi,
From the document I'm unclear if I can deploy Wazuh automatically via the forward as the document says:

"When using ossec-authd, be sure to add a firewall exception for agents to access port 1515/tcp on the Wazuh manager node by running so-allow and choosing the r option."

Do I need to allow all wazuh agents that I want to automatically deploy to have direct access to the master? As I'm guessing the master hosts the wazuh manager.

**Comment by dougburks** [comment, marked as answer]
Each Security Onion node includes its own Wazuh manager.
